### PR TITLE
Add NatSpec to Contract type and mention in README

### PR DESCRIPTION
**Pull request by BjMrq**
Add natSpec (as a string) to Contract interface and mention in README

### CODE REVIEW

**Review by mok0230 — APPROVED** [review]
lgtm FYI @spalladino this enables setting natspec on a contract even without sourcify (only available via API as of now)

File: packages/admin/README.md [hunk 1]
```diff
@@ -126,14 +126,15 @@ const contract = {
 await client.proposeUpgrade({ newImplementation }, contract);
 ```
 
-Alternatively, you can add any contract explicitly by using the `addContract` method, and setting network, address, name, and ABI. The same method can be used to update the contract's name or ABI.
+Alternatively, you can add any contract explicitly by using the `addContract` method, and setting network, address, name, natspec and ABI. The same method can be used to update the contract's name or ABI.
```
Comment (mverzilli) [inline packages/admin/README.md:129]: Just double checking, aren't we allowing them to update natspec? (if so we should also mention it in the last sentence here)

**Review by mverzilli — APPROVED** [review]

File: packages/admin/README.md (hunk 1 above)
Comment (BjMrq) [reply packages/admin/README.md:129]: oh actually I just checked and we do not as of now

Comment (mok0230) [reply packages/admin/README.md:129]: @BjMrq could you please capture updating natspec in a separate issue? Otherwise lgtm

**Pull request merged**